app/helpers/fhir_helper.rb

```diff
@@ -198,7 +198,7 @@ def to_us_core_race(white, black_or_african_american, american_indian_or_alaska_
   # Return a boolean indicating if the given race code is present on the given FHIR::Patient.
   def race_code?(patient, code)
     url = 'us-core-race'
-    patient&.extension&.select { |e| e.url.include?(url) }&.first&.extension&.select { |e| e.url == 'ombCategory' }&.first&.valueCoding&.code == code
+    patient&.extension&.select { |e| e.url.include?(url) }&.first&.extension&.select { |e| e.url == 'ombCategory' }&.any?{ |e| e&.valueCoding&.code == code }
```

Comment:
We use rubocop to do linting, and it is mad at this line. Could you update this line to be like this:
```suggestion
    patient&.extension&.select { |e| e.url.include?(url) }&.first&.extension&.select { |e| e.url == 'ombCategory' }&.any? { |e| e&.valueCoding&.code == code }
```
All it does is add a space after `any?`, but it keeps the linter happy.

Reply:
Good with me, thanks!